doc/build/dts/bindings.rst

```diff
@@ -1048,6 +1025,85 @@ The following general rules apply to vendor prefixes in :ref:`compatible
   specific drivers handling the nodes, it's usually best to use ``zephyr`` as
   the vendor prefix. See :ref:`dt_vendor_zephyr` for examples.
 
+.. _dt-bindings-default-rules:
+
+Rules for default values
+========================
+
+In any case where ``default:`` is used in a devicetree binding, the
+``description:`` for that property **must** explain *why* the value was
+selected and any conditions that would make it necessary to provide a different
+value. Additionally, if changing one property would require changing another to
+create a consistent configuration, then those properties should be made
+required.
+
+There is no need to document the default value itself; this is already present
+in the :ref:`devicetree_binding_index` output.
+
+There is a risk in using ``default:`` when the value in the binding may be
+incorrect for a particular board or hardware configuration.  For example,
+defaulting the capacity of the connected power cell in a charging IC binding
+is likely to be incorrect.  For such properties it's better to make the
+property ``required: true``, forcing the user to make an explicit choice.
+
+Driver developers should use their best judgment as to whether a value can be
+safely defaulted. Candidates for default values include:
+
+- delays that would be different only under unusual conditions
+  (such as intervening hardware)
+- configuration for devices that have a standard initial configuration (such as
+  a USB audio headset)
+- defaults which match the vendor-specified power-on reset value
+  (as long as they are independent from other properties)
+
+Power-on reset values may be used for defaults as long as they're independent.
```

Comment:
These two sentences are redundant.

I'm also not sure we should qualify with "as long as they're independent". I think power-on reset is sufficient and requiring independence may confuse people. Can you give an example of some properties that you'd advise against defining a default due to a dependency?